bin/llvm-kompile-clang

```diff
@@ -244,6 +244,7 @@ if $link; then
     "${flags[@]}" \
     "${all_libraries[@]}" \
     -I "$(dirname "$0")"/../include/kllvm \
+    -I "$(dirname "$0")"/../include \
```

Comment:
I can answer this. We need to be able to import AST.h from runtime/header.h now, however, runtime/header.h is included from two different contexts: first, from C++ files in the backend itself compiled by CMake, and second, from C++ files that are compiled and linked by llvm-kompile. In the former case, we were already qualifying the kllvm includes with `kllvm`, but in the latter we were not. Thus, there was no way to include this file such that both code paths would compile. We add this include path so that we can qualify the include in runtime/header.h with `kllvm` and have it compile on both code paths.